### ISSUE TITLE
Can set save path to sdcard?

### ISSUE THREAD

**Issue by GHolk** [ultranity/Pix-EzViewer#86]
能把圖片下載到記憶卡裡嗎？
選擇保存路徑的對話框沒能切到記憶卡，似乎沒這個功能？
但聽說存取記憶卡比較麻煩，我可能寫不了。

**Comment by GHolk** [comment]
yes it can, without extra app permision. We just need a new ui for this.

## Config it with root
force stop the app and clean the cache.
su to root, edit `/data/data/com.perol.asdpl.play.pixivez.libre/shared_prefs/com.perol.asdpl.play.pixivez.libre_preferences.xml`, and change the storepath1 to the sdcard path.

sdcard path should look like this: `/storage/89AB-EF01/my-pixez`

**Comment by ultranity** [comment]
我缺少支持sdcard的设备来调试这个feature :( 

如果权限支持只是需要选择，可以改造下folderChooser
但是对非root用户，在高版本android上面估计会有些兼容性适配问题

**Comment by Trung0246** [comment]
Hi, I can't set storage path to network storage either. App in question:

https://github.com/wa2c/cifs-documents-provider
https://play.google.com/store/apps/details?id=dev.rocli.android.webdav

At this point maybe just use Storage Access Framework to properly allows picking arbitrary folder instead of just current bespoke custom file picker.
